sdk/storage/AzureStorageBlob/Source/TransferManager.swift

```diff
@@ -48,13 +48,9 @@ internal protocol TransferManager {
 
     func add(transfer: TransferImpl)
     func cancel(transfer: TransferImpl)
-    func cancelAll(withRestorationId: String?)
```

Comment:
Why are these being removed?

Reply:
As part of this, a lot of the functionality we used to have in the manager is now part of either the Transfer object itself or the TransferCollection. For example, `transfer.cancel()` and `transferCollection.cancelAll()`. Thus the TransferManager protocol no longer needed to include the `*All` methods. The concrete implementation of the transfer manager still does need some implementations of them though.